couchbase/datadog_checks/couchbase/couchbase.py

```diff
@@ -178,7 +181,7 @@ def _get_stats(self, url):
 
     def check(self, instance):
         instance_state = self._instance_states[hash_mutable(instance)]
-
+        sync_gateway_url = instance.get("sync_gateway_url")
         server = instance.get('server', None)
         if server is None:
             raise Exception("The server must be specified")
```

Comment:
Would these be better in the Check `init` method, then can just check the instance variables and change this method signature to `check(self, _)`?